src/dlgprefreplaygain.cpp

```diff
@@ -108,8 +108,7 @@ void DlgPrefReplayGain::slotUpdateReplayGainBoost() {
 
 void DlgPrefReplayGain::setLabelCurrentReplayGainBoost(int value) {
     LabelCurrentReplayGainBoost->setText(
-            QString(tr("%1 dB (average %2 dB)")).arg(
-                  QString().sprintf("%+d", value), QString::number(value - 14)));
+            QString(tr("Adjust by %1 dB")).arg(QString().sprintf("%+d", value)));
```

Comment:
The -14db number makes no sense if users don't have a background in loudness theory.  They may think that they should apply a +14db adjustment to their tracks, which will cause massive clipping.  It is not useful to know about the -14db value when setting up mixxx, so I don't think we should mention it.  However it is useful to know that a +6db adjustment will restore the loudness of most tracks.